web/tests/api-spec.js

```diff
@@ -4,13 +4,15 @@ describe('api service', function () {
         fixture.setBase('tests/api-description');
 
         inject(function ($http) {
-            const MOCK_PROJECT_SETTINGS = {
-                API_ROOT: 'localhost:8000'
+            const mockProjectConfig = {
+                getSettings:() => ({
+                    apiRoot: 'localhost:8000'
+                })
```

Comment:
Still, writing the function out in long form is clearer. I don't think there can be any argument with that?

I'm not saying you can't understand these things once you are familiar with them, but it makes the barrier to entry higher. 
If you are not using the `this` binding, the advantage of arrows is shorter code. 
The disadvantage is being less readable.
If given this choice in a team setting where people are still learning and not everyone is a JS expert, IMO the clear choice is to choose the more readable way?

There are many things in our style guide where we recommend to write for readability rather than using code shortcuts. e.g. for the the same reason we try to avoid terniary operators instead of `if..then` because it is harder to understand.